arm-billing/2017-02-27-preview/swagger/billing.json

```diff
@@ -0,0 +1,399 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2017-02-27-preview",
+    "title": "BillingClient",
+    "description": "Billing client provides access to billing resources for Azure Web-Direct subscrions. Other subscription types which were not purchased directly through the Azure web portal are not supported through this preview API."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Billing/invoices": {
+      "get": {
+        "tags": [
+          "Invoices"
+        ],
+        "operationId": "Invoices_List",
+        "description": "Lists the available invoices for a subscritpion in reverse chronological order beginning with the most recent invoice. In preview, invoices are available via this API only for invoice periods which end December 1, 2016 or later",
```

Comment:
vscode is a very good tool for editing swaggers. 
It should provide you intellisense and basic validation right off the bat, because we have added schema support over [here](https://github.com/Azure/azure-rest-api-specs/blob/master/.vscode/settings.json).

You can also install a spell checker extension from the awesome vscode gallery ;).